src/templates/podcast.tsx

```diff
@@ -180,6 +190,8 @@ const Podcast = ({
                 image={image}
                 publishDate={publishDate}
                 title={title}
+                tags={tagList}
```

Comment:
For this component, I noticed somewhere we renamed `title` prop to `articleTitle`. Can we swap that (line 166) to `title` to show up in the tab?

Reply:
Similar comment for videos